### PR TITLE
clarify signed blob transaction hashing

### PR DESCRIPTION
**Pull request by roberto-bayardo**

### CODE REVIEW

File: EIPS/eip-4844.md [hunk 1]
```diff
@@ -174,6 +174,13 @@ def get_origin(tx: SignedBlobTransaction) -> Address:
     return ecrecover(tx_hash(tx), int(sig.y_parity)+27, sig.r, sig.s)
 ```
 
+The transaction hash of a signed blob transaction should be computed per [`EIP-2718`](./eip-2718.md):
+
+```python
+def signed_tx_hash(tx: SignedBlobTransaction) -> Bytes32:
+    return keccak256(BLOB_TX_TYPE + ssz.serialize(tx))
+```
```
Comment (MicahZoltu) [inline EIPS/eip-4844.md:182]:
````suggestion
```
````

File: EIPS/eip-4844.md [hunk 2]
```diff
@@ -174,6 +174,13 @@ def get_origin(tx: SignedBlobTransaction) -> Address:
     return ecrecover(tx_hash(tx), int(sig.y_parity)+27, sig.r, sig.s)
 ```
 
+The transaction hash of a signed blob transaction should be computed per [`EIP-2718`](./eip-2718.md):
```
Comment (MicahZoltu) [inline EIPS/eip-4844.md:177]: EIP-2718 doesn't specify how transaction hashes are calculated.  I *believe* that transaction hashes are the `keccak256` of the bytes in the transaction trie whose root is in the block header, so if the below aligns with that then it is correct.
```suggestion
The transaction hash of a signed blob transaction should be computed as:
```

**Comment by roberto-bayardo** [comment]
See also https://github.com/ethereum/EIPs/pull/6241 which I am keeping separate from this since it's a breaking change rather than a clarification.

**Comment by protolambda** [comment]
> See also https://github.com/ethereum/EIPs/pull/6241 which I am keeping separate from this since it's a breaking change rather than a clarification.

Left a comment there about RLP<>SSZ and hashing behavior which I think is relevant to both PRs.

**Comment by roberto-bayardo** [comment]
It is possible to get these PRs merged by the sync tomorrow @protolambda ?  The code changes to the 4844-geth are already subnmitted.

**Review by protolambda — APPROVED** [review]

**Pull request merged**